connectors/sources/azure_blob_storage.py

```diff
@@ -212,6 +216,7 @@ async def get_container(self, container_list):
         Yields:
             dictionary: Container document with name & metadata
         """
+        self._logger.debug("Fetching containers")
```

Comment:
in `get_blob` we have log level `INFO`, maybe worth changing log level `INFO` here as well to keep it consistent?